src/Uno.UI/UI/Xaml/Controls/ComboBox/ComboBox.cs

```diff
@@ -379,11 +413,6 @@ protected override void OnIsEnabledChanged(bool oldValue, bool newValue)
 				DropDownOpened?.Invoke(this, newIsDropDownOpen);
 
 				RestoreSelectedItem();
-
-				if (SelectedItem != null)
```

Comment:
It doesn't seem like this ever worked: ListViewBase.ScrollIntoView() is only implemented for Android and iOS, and even there, only for the virtualizing panel. But ComboBox doesn't use the virtualizing panel on Android and iOS. So this would not ever have actually done anything.